db/seeds.rb

```diff
@@ -19,8 +10,20 @@
   password_confirmation: 'password'
```

Comment:
@pkarman updated seeds to look at an ENV var for password. Going to add basic auth in a different PR if that is OK with you